src/ProjectOrigin.WalletSystem.IntegrationTests/REST/CertificatesControllerTests.cs

```diff
@@ -100,6 +86,63 @@ public async Task Test_AggregateCertificates(string timezone, long[] values, Cer
         resultList.Select(x => x.Quantity).Should().ContainInOrder(values);
     }
 
+    [Fact]
+    public async Task Certificates_Cursor()
+    {
+        var issuestartDate = new DateTimeOffset(2020, 6, 1, 12, 0, 0, TimeSpan.Zero);
+        var issueEndDate = new DateTimeOffset(2020, 6, 30, 12, 0, 0, TimeSpan.Zero);
+        var queryStartDate = new DateTimeOffset(2020, 6, 8, 12, 0, 0, TimeSpan.Zero);
+        var queryEndDate = new DateTimeOffset(2020, 6, 10, 12, 0, 0, TimeSpan.Zero);
+
+        var subject = _fixture.Create<string>();
+        var controller = new CertificatesController
+        {
+            ControllerContext = CreateContextWithUser(subject)
+        };
+
+        var wallet = await _dbFixture.CreateWallet(subject);
+        var endpoint = await _dbFixture.CreateWalletEndpoint(wallet);
+
+        await CreateCertificates(issuestartDate, issueEndDate, endpoint);
+
+        var result = await controller.GetCertificates(
+            _unitOfWork,
+            new GetCertificatesQueryParametersCursor
+            {
+                Limit = 10,
+                Start = queryStartDate.ToUnixTimeSeconds(),
+                End = queryEndDate.ToUnixTimeSeconds(),
+                UpdatedSince = DateTimeOffset.UtcNow.AddHours(-1).ToUnixTimeSeconds()
```

Comment:
So you want 10 certificates between start and end that is updated since one hour ago? What happens if nothing is updated since then?